lms/djangoapps/instructor/enrollment.py

```diff
@@ -158,7 +160,10 @@ def enroll_email(course_id, student_email, auto_enroll=False, email_students=Fal
             email_params['message_type'] = 'enrolled_enroll'
             email_params['email_address'] = student_email
             email_params['full_name'] = previous_state.full_name
-            send_mail_to_student(student_email, email_params, language=language)
+            if is_testing_environment():
+                send_mail_to_student(student_email, email_params, language=language)
+            else:
+                compose_and_send_adg_course_enrollment_invitation_email(student_email, email_params)
```

Comment:
These lines are duplicated from line 175-178. Maybe place them in a function to remove duplication?

Reply:
Same as discussed [above](https://github.com/OmnipreneurshipAcademy/edx-platform/pull/23#discussion_r516731629)